mephisto/server/blueprints/acute_eval/webapp/src/components/core_components.jsx

```diff
@@ -232,15 +232,15 @@ class EvalResponse extends React.Component {
   }
 
   checkValidData() {
+    let response_data = {
+      speakerChoice: this.state.speakerChoice,
+      textReason: this.state.textReason,
+    };
     if (this.state.speakerChoice !== "" && this.state.textReason.length > 4) {
-      let response_data = {
-        speakerChoice: this.state.speakerChoice,
-        textReason: this.state.textReason,
-      };
       this.props.onValidDataChange(true, response_data);
       return;
```

Comment:
tiniest of nits - i'd probably just get rid of this `return` and wrap the `onValidDataChange` call below in an `else` block